app/services/acceptable_verified_credential_adoption_service.rb

```diff
@@ -77,7 +78,29 @@ def verified_credential_at?
     user_avc&.acceptable_verified_credential_at
   end
 
-  def log_results(conversion_type)
+  def log_conversion_type_results(conversion_type)
     StatsD.increment("#{STATS_KEY}.#{conversion_type}.#{credential_type}")
   end
+
+  def record_adoption_email_trigger_event
+    CredentialAdoptionEmailRecord.create(
+      icn: user.icn,
+      email_address: user.email,
+      email_template_id: REACTIVATION_TEMPLATE_ID,
+      email_triggered_at: DateTime.now
+    )
+  end
+
+  def check_for_email_adoption_records
+    CredentialAdoptionEmailRecord.where('email_triggered_at > ?', DateTime.now.days_ago(7))
```

Comment:
If they logged in again using the legacy credential seven (7) days after the first email was triggered, yes they would then trigger the email to be sent again.

The seven (7) day timeline was somewhat arbitrarily set since we only planned to have this feature enabled for one (1) week at most.  

In the [follow-up ticket/PR](https://github.com/department-of-veterans-affairs/vets-api/pull/13304) to this one, we outline the one (1) week experiment timeline.

Reply:
Sounds good, just checking that's the intention

Reply:
Oh wait, if the feature is planned for 7 days only, would be it simpler just to check and see if the record exists and to add a timeline at some future point?

Reply:
@bosawt It would have been simpler, yes :)  If this is a blocking change please let me know, otherwise I'll add it to the follow-up PR after this one (which will be going up as soon as this PR is approved and merged in).